### PR TITLE
Init TTL and hop_limit in Echo response

### PR DESCRIPTION
**Pull request by dpino**
Fixes https://github.com/Igalia/snabb/issues/1069

### CODE REVIEW

**Comment by wingo** [comment]
Looks good.  In some future we should make this default TTL configurable, but it is good as is.  Would you mind adding an end-to-end test?

File: src/program/lwaftr/tests/end-to-end/test_env.sh [hunk 1]
```diff
@@ -385,18 +385,30 @@ TEST_DATA=(
 "no_icmp_with_filters_drop.conf" "tcp-frominet-trafficclass.pcap" "" "" ""
 "in-1p-ipv4-out-0p-drop.lua"
 
+# Ping to lwAFTR (IPv4).
+
 "ICMP Echo to AFTR (IPv4)"
 "no_icmp.conf" "ping-v4.pcap" "" "ping-v4-reply.pcap" ""
```
Comment (wingo) [inline src/program/lwaftr/tests/end-to-end/test_env.sh:391]: Question -- why were our existing ping tests not testing this behavior?  After the behavior was changed, why didn't the tests start failing?

Comment (dpino) [reply src/program/lwaftr/tests/end-to-end/test_env.sh:391]: Before the reply used the same TTL as the request. In our tests the ICMP echo requests ("ping-v4.pcap") have TTL=64 (which is now set as default value in replies). After the change, the replies didn't actually varied (there are in both cases ICMP echo response with TTL=64).

**Comment by wingo** [comment]
Where did the tests go?  I could have sworn you added some end-to-end tests.

**Comment by dpino** [comment]
I think I overwrote the PR by mistake, removing the tests. Luckily I still had the changes on a local branch.

**Review by wingo — APPROVED** [review]

**Pull request merged**